geoportal/geoportailv3_geoportal/static-ngeo/js/apps/MainController.js

```diff
@@ -941,6 +948,15 @@ const MainController = function(
   this.loadThemes_().then(() => {
     this.appThemes_.getBgLayers(this.map_).then(
           bgLayers => {
+
+            const hillshadeConfig = new URLSearchParams(window.location.search).get('hillshade');
+            if (hillshadeConfig) {
+              this.hillshadeStylingData = JSON.parse(hillshadeConfig);
+              setTimeout(() => {
+                this.getSetHillshadeVisible(JSON.parse(hillshadeConfig)[0]['visible']);
+              }, 750); // without it, get "Style is not done loading" error
```

Comment:
This is brittle and will fail if the style gets longer to load (slow network, slow device, ...).
Hillshade should be part of the style configuration.